addons/test_event_full/tests/test_event_mail.py

```diff
@@ -125,3 +128,38 @@ def test_event_mail_before_trigger_sent_count(self):
             'Wrong Emails Sent Count! Probably emails sent to unconfirmed attendees were not included into the Sent Count')
         self.assertEqual(mail_scheduler.filtered(lambda r: r.notification_type == 'sms').mail_count_done, 2,
             'Wrong SMS Sent Count! Probably SMS sent to unconfirmed attendees were not included into the Sent Count')
+
+
+class TestEventSaleMailSchedule(TestEventFullCommon):
```

Comment:
Can't we add it in previous class ? Seems in the end we could probably have a single class, but for stable maybe just add it in the previous one ?

Reply:
Indeed, we could. I used `TestEventFullCommon` because there was already the needed variables in the setup, in this case a sale order (`self.customer_so`).
Should I still move it to `TestEventSmsMailSchedule` and create the needed sale order in the test?

Reply:
Mmmh, in the end it is ok. We will cleanup in master when we have some time :)